modules/core/src/main/java/org/apache/ignite/events/CacheConsistencyViolationEvent.java

```diff
@@ -67,39 +68,58 @@
     private static final long serialVersionUID = 0L;
 
     /** Represents original values of entries.*/
-    final Map<Object, Map<ClusterNode, EntryInfo>> entries;
+    final Map<?, Map<ClusterNode, EntryInfo>> entries;
+
+    /** Fixed entries. */
+    final Map<?, ?> fixed;
 
     /** Cache name. */
     final String cacheName;
 
+    /** Strategy. */
+    final ReadRepairStrategy strategy;
+
     /**
      * Creates a new instance of CacheConsistencyViolationEvent.
-     *
-     * @param cacheName Cache name.
+     *  @param cacheName Cache name.
      * @param node Local node.
      * @param msg Event message.
      * @param entries Collection of original entries.
+     * @param fixed Collection of fixed entries.
+     * @param strategy
      */
     public CacheConsistencyViolationEvent(
         String cacheName,
         ClusterNode node,
         String msg,
-        Map<Object, Map<ClusterNode, EntryInfo>> entries) {
+        Map<?, Map<ClusterNode, EntryInfo>> entries,
+        Map<?, ?> fixed, ReadRepairStrategy strategy) {
         super(node, msg, EVT_CONSISTENCY_VIOLATION);
 
         this.cacheName = cacheName;
         this.entries = entries;
+        this.fixed = fixed;
+        this.strategy = strategy;
     }
 
     /**
      * Returns a mapping of keys to a collection of original entries.
      *
      * @return Collection of original entries.
      */
-    public Map<Object, Map<ClusterNode, EntryInfo>> getEntries() {
+    public Map<?, Map<ClusterNode, EntryInfo>> getEntries() {
```

Comment:
Looks like "?" is not required in this class at all.
Have no clue why it was required before, but this was made because of compilation issues.
Thanks for pointing this.